packages/cli/lib/SpinniesManager.js

```diff
@@ -1,104 +1,346 @@
-const Spinnies = require('spinnies');
+'use strict';
```

Comment:
This is mostly just a 1:1 port from the spinnies library [here](https://github.com/jbcarpanelli/spinnies/blob/master/index.js)